### ISSUE TITLE
White noise, audio distortion and eventually silence as I play a match.

### ISSUE THREAD

**Issue by gumacahin** [ValveSoftware/Dota-2#549]
This started with the recent patch. Ten to fifteen minutes into a match I get a noticeable white noise. Ten more minutes and all audio gets a little distorted. Ten more minutes and it's just noise. Ten more minutes and it's silence.

Model Name: Mac mini
  Model Identifier: Macmini7,1
  Processor Name:   Intel Core i5
  Processor Speed:  2.6 GHz
  Number of Processors: 1
  Total Number of Cores:    2
  L2 Cache (per Core):  256 KB
  L3 Cache: 3 MB
  Memory:   8 GB

Scarlett 2i2 USB:

  Default Input Device: Yes
  Default Output Device:    Yes
  Default System Output Device: Yes
  Input Channels:   2
  Manufacturer: Focusrite
  Output Channels:  2
  Current SampleRate:   44100
  Transport:    USB

**Comment by gdrewb-valve** [comment]
Is this still happening?  Can you post your full system information from Steam?

**Comment by gumacahin** [comment]
Yes, it still does. It's even worse now. Previously, my fix is to temporarily switch to the on-board audio then switch back to the 2i2 to clear the noise. This fix no longer works. When in a match I have to switch to on-board audio if I want to enjoy the match noise free. When the match is over I'd have to restart Dota2 to clear the noise.

`Processor Information:
    Model:  Macmini7,1
    Vendor:  GenuineIntel
    CPU Family:  0x6
    CPU Model:  0x45
    CPU Stepping:  0x1
    CPU Type:  0x0
    Speed:  2600 Mhz
    4 logical processors
    2 physical processors
    HyperThreading:  Supported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported

Network Information:
    Network Speed:  

Operating System Version:
    MacOS 10.11.3 (64 bit)

Video Card:
    Driver:  Intel Iris

```
Driver Version:  1.2
Driver Date: 0  0
OpenGL Version: 2.1
Desktop Color Depth: 32 bits per pixel
Monitor Refresh Rate: 0 Hz
VendorID:  0x8086
DeviceID:  0xa2e
Number of Monitors:  1
Number of Logical Video Cards:  1
Primary Display Resolution:  1920 x 1200
Desktop Resolution: 1920 x 1200
Primary Display Size: 20.39" x 12.76"  (24.02" diag)
                                        51.8cm x 32.4cm  (61.0cm diag)
Primary VRAM: 1536 MB
```

Memory:
    RAM:  8192 Mb

Miscellaneous:
    UI Language:  English
    Microphone:  Not set
    Media Type:  DVD
    Total Hard Disk Space Available:  1192189 Mb
    Largest Free Hard Disk Block:  320283 Mb

Installed software:
    Accessibility Inspector
    Acrobat Distiller
    Adobe Acrobat
    Adobe After Effects CC 2015
    Adobe After Effects Render Engine
    Adobe Audition CC 2015
    Adobe Character Animator (Preview)
    Adobe Flash CC 2015
    Adobe Illustrator CC 2015
    Adobe Lightroom
    Adobe Media Encoder CC 2015
    Adobe Photoshop CC 2015
    Adobe Photoshop Lightroom 5
    Adobe Premiere Pro CC 2015
    Analyze Documents
    Android File Transfer
    Android Studio
    App Store
    Application Loader
    Automator
    Balsamiq Mockups 3
    BibDesk
    Calendar
    Canon IJ On-screen Manual
    Canon IJ Scan Utility2
    Canon IJ ScanGear Starter EX
    Canon Inkjet Printer/Scanner/Fax Extended Survey Program
    Canon Quick Menu
    CINEMA 4D Lite
    CINEMA 4D QT Agent
    CineRenderAE
    CineWare
    Contact Sheets
    Contacts
    DrvPermanent
    EOS Lens Registration Tool
    EOS Utility
    EOS Utility 2
    EOS Web Service Registration Tool
    Evernote
    Excalibur
    Export Flash Animation
    FaceTime
    FileMerge
    FileZilla
    Firefox
    Flash Player
    Flash Player Debugger
    Game Center
    Google Chrome
    Google Drive
    Google Earth
    Google Search
    iBooks
    Image Capture
    Instruments
    Keynote
    LaTeXiT
    Launchpad
    Lightshot Screenshot
    Logitech Options
    MacVim
    Make Calendar
    Maps
    Messages
    Minecraft
    MiniTerm
    Mission Control
    MKVtools
    mocha AE CC
    monitor
    MySQLWorkbench
    naib
    Notes
    Numbers
    OBS
    Pages
    Photos
    PhotoStitch
    PhotoStitch Launcher
    ReadMe(AutoUpdateService)
    ReadMe(EOS Utility)
    ReadMe(Image Handling Library)
    ReadMe(PhotoStitch)
    Reminders
    SA Color Finesse 3 UI
    Skype
    Steam
    Stickies
    SyphonInject
    TabletSettings
    TeamViewer
    Template
    TeX Live Utility
    TeXShop
    Transmission
    Uninstall PenTablet Driver
    UpdateChecker
    Viber
    Viewer
    VLC
    Web Gallery
    Xcode
    YouTube

`

**Comment by gdrewb-valve** [comment]
Thanks.  Reborn uses SDL for audio so I'll pass this on to the SDL folks and see if they know anything about it.
